server/conf/application.dev-browser-tests.conf

```diff
@@ -31,3 +31,8 @@ name_suffix_dropdown_enabled = true
 # In the test environment we don't need to have the jobs running at the
 # default 5 second interval
 durable_jobs.poll_interval_seconds = 3600
+
+session_inactivity_warning_threshold_minutes = 10 # Warn the user 10 mins before the inactivity timeout
```

Comment:
question (non-blocking): It doesn't hurt to set these here, but since they're not directly used in your test (unless I'm missing something) I think you don't need them. I'm pretty sure the test conf file will inherit from the dev conf file which inherits from the prod conf file.

Reply:
Unfortunately we need this to order all warnings happen before the timeouts as it makes the tests easier to write(trigger all warnings first and then test the timeout). If the timeout event happen before warning(which is what happens in default), It is not possible to test warning by our current approach of advancing time.